src/DataFixtures/EventFixture.php

```diff
@@ -45,6 +48,8 @@ public function load(ObjectManager $manager): void
             ->setLocation($this->getReference(LocationFixture::ITKDEV))
             ->addTag($this->getReference(TagsFixtures::CONCERT))
             ->addTag($this->getReference(TagsFixtures::AROS))
+            ->setEditable(true)
```

Comment:
Do we read events from a feed when loading fixtures? Otherwise, consider making a single fixture event non-editable.

Reply:
Yes, when you process the loaded feed fixtures you get 40 events.